docs/setup/ubuntu2204.md

```diff
@@ -0,0 +1,77 @@
+# Install ROS 2 Humble with Create 3 Messages on an Ubuntu 22.04 Machine
+
+## Before you start
+If you are running Ubuntu[^1] 22.04 natively on your machine, there is no extra setup required.
```

Comment:
it seems unclear to me which steps can be skipped if you already have ubuntu 22.04

Reply:
None of them! Good point, this line is a holdover from the original Pi directions from Galactic.